flake.nix

```diff
@@ -43,7 +43,8 @@
       };
     };
     # Question: move this to common?
-    services-flake.url = "github:juspay/services-flake";
+    services-flake.url = "github:roman-bodavskiy/services-flake/clickhouse";
+    # services-flake.url = "github:juspay/services-flake";
```

Comment:
should be
`services-flake.url = "github:juspay/services-flake"`
because flakes PR already merged